sbol2/config.py

```diff
@@ -98,6 +99,21 @@ class Config:
     through the setOptions and getOptions methods.
     """
 
+    # Dictionary which holds builder functions for SBOL objects and which can
+    # be extended with extension classes. The dictionary is populated in
+    # document.py
+    SBOL_DATA_MODEL_REGISTER = {}
+
+    def register_extension_class(cls, type_uri):
+        """Register an extension class and its namespace, so custom data
```

Comment:
The use of a class here is confusing. If we use the value in `SBOL_DATA_MODEL_REGISTER` as a class somewhere, then it's fine, and we should document the argument as being a class, not a class constructor. If on the other hand we really need a builder function in the `cls` argument, then we should say that. A builder function would be a no-arg function (I think) that constructs and instance during document load. As it stands now, we've got our foot in both camps calling it a class and a class constructor.

Reply:
I've update the documentation. Hopefully it is clearer now.